commands/peers/commands.go

```diff
@@ -27,6 +27,20 @@ func (c *Command) Routes() rest.Routes {
 			HandlerFunc: getPeersHandler,
 		},
 		rest.Route{
+			Name:        "EtcdHealthPeer",
+			Method:      "GET",
+			Pattern:     "/peers/{peerid}/etcdhealth",
```

Comment:
On a second thought, in future, if etcd isn't run on all nodes, then probably we should keep this distinction between `/etcdhealth` and `/status` endpoints.